eng/pipelines/common/global-build-job.yml

```diff
@@ -154,10 +145,7 @@ jobs:
 
     - checkout: self
       clean: true
-      # If running in source build mode, a git stash will be used for the inner clone. Avoid setting a fetch depth,
-      # as a stash of a shallow cloned repo is not currently supported.
-      ${{ if ne(parameters.isSourceBuild, true) }}:
-        fetchDepth: $(checkoutFetchDepth)
+      fetchDepth: $(checkoutFetchDepth)
```

Comment:
Who sets this for SB?

Reply:
SourceBuild has never had this set (due to the comment above).